lib/core/BlogPostLayout.js

```diff
@@ -24,7 +24,7 @@ class BlogPostLayout extends React.Component {
       this.props.config.facebookComments && (
         <div className="blogSocialSectionItem">
           <div
-            className="fb-comments"
+            className="fb-comments" // Facebook SDK require it to be 'fb-comments'
```

Comment:
Please make sure this doesn't show up in the output. I don't remember being able to write comments in JSX like that. I had to do 

```
{ /* Facebook SDK require it to be 'fb-comments */ }
```

Reply:
ah that's true. I almost forgot 😢